registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java

```diff
@@ -1071,7 +1080,7 @@ private void addFirstOrderAddress(ComboBox<GenericDto> location, int id, String
 			location.getItems().clear();
 			try {
 				List<GenericDto> locations = null;
-				locations = masterSync.findLocationByHierarchyCode(id, languageType);
+				locations = sortLocationListAsc(masterSync.findLocationByHierarchyCode(id, languageType));

```

Comment:
better to do a null check for masterSync.findLocationByHierarchyCode(...)

Reply:
Added null check inside sortLocationListAsc function